### PR TITLE
Update new algorithm service doc

### PR DESCRIPTION
**Pull request by andreyvelich**
We should update our doc for the new Algorithm service since our CI has been changed.

/cc @gaocegege @johnugeorge @PatrickXYS

### CODE REVIEW

File: docs/new-algorithm-service.md [hunk 1]
```diff
@@ -138,106 +144,91 @@ if __name__ == '__main__':
...
 
 #### E2E Test (Optional)
 
 E2e tests help Katib verify that the algorithm works well.
-To add a e2e test for the new algorithm, in [test/scripts/v1beta1](../test/scripts/v1beta1) you need to:
+Follow bellow steps to add your algorithm (Suggestion) to the Katib CI
+(replace `<name>` with your Suggestion name):
 
-1. Create a new Experiment yaml file in [examples/v1beta1](../examples/v1beta1) with the new algorithm.
+1. Submit a PR to add a new registry to the AWS
```
Comment (PatrickXYS) [inline docs/new-algorithm-service.md:155]: Can we rephrase it to be 

```
Submit a PR to add a new ECR private registry to the AWS
``

Comment (andreyvelich) [reply docs/new-algorithm-service.md]: Thanks @PatrickXYS! Done.

**Comment by andreyvelich** [comment]
Hold this PR until we merge: https://github.com/kubeflow/katib/pull/1464.
It requires some additional changes in this doc.

**Comment by andreyvelich** [comment]
This PR is ready.
Please take take a look @PatrickXYS @gaocegege @johnugeorge 
/hold cancel

**Review by PatrickXYS — APPROVED** [review]
LGTM

Thanks for updates @andreyvelich !
